wagtail/images/templates/wagtailimages/images/url_generator.html

```diff
@@ -17,20 +29,20 @@
             {% endfield_row %}
         </form>
 
-        {% trans "URL" as heading %}
-        {% panel id="url" icon="link" heading=heading %}
-            <textarea id="result-url" rows="1"></textarea>
-        {% endpanel %}
-
         {% trans "Preview" as heading %}
         {% panel id="preview" icon="view" heading=heading %}
             <div>
-                <div class="loading-mask inline-block">
-                    <img class="preview" src="" alt="{% trans 'Preview' %}" />
+                <div class="loading-mask">
```

Comment:
```suggestion
                <div class="loading-mask w-block w-border-2 w-border-border-furniture w-overflow-scroll w-p-4 w-rounded-sm">
```

We can get pretty far here with Tailwind, we cannot do the max-height item for now but the rest can be done this way.